std_msgs/Header header
PointDetect left1
PointDetect right1
PointDetect left2
PointDetect right2
